documentation/docs/pipelines/abapEnvironment/stages/Test.md

```diff
@@ -0,0 +1,140 @@
+# Test
+
+This stage contains of two sub stages, `ATC` and `AUnit` which can execute ATC checks and AUnit test runs in parallel on an SAP BTP ABAP Environment systen.
+By default this stage will not run any of the two sub stages `ATC` and `AUnit` if they are not configured. Please keep in mind that the `ATC` and `AUnit` stages need to be configured independently.
+Below sections contain more information on the usage and configuration of the `ATC` and `AUnit` stages.
+
+## ATC
+
+In this stage, ATC checks can be executed using abapEnvironmentRunATCCheck. The step can receive software components or packages. The results are returned in the checkstyle format. With the use of a pipeline extension, quality gates can be configured (see [step documentation](https://sap.github.io/jenkins-library/steps/abapEnvironmentRunATCCheck/) or the "Extensions" section in the [configuration](../configuration.md)).
+
+### Steps
+
+The following steps are executed in this stage:
+
+- [abapEnvironmentRunATCCheck](../../../steps/abapEnvironmentRunATCCheck.md)
+
+### Stage Parameters
+
+There are no specifc stage parameters.
+
+### Stage Activation
+
+This stage will be active, if the stage configuration in the `config.yml` contains entries for this stage.
+Please keep in mind that the configuration for the `ATC` stage needs to be subsequent to the `Test` stages hirarchy.
```

Comment:
```suggestion
```
The "ATC" stage activation does not depend on the "Test" stage. Also, there is no configuration for the "Test" stage. To me it is not clear what this sentence means - therefore I would leave it out.